index.bs

```diff
@@ -1040,6 +1185,111 @@ spec: workers; urlPrefix: https://www.w3.org/TR/workers/#
       {{Screen}}.
     </p>
   </section>
+
+  <section>
+    <h3 id='video-display-capabilities'>Video Display Capabilities</h3>
+
+    <section>
+      <h4 id='video-display-configuration'>VideoDisplayConfiguration</h4>
+
+      <pre class='idl'>
+        dictionary VideoDisplayConfiguration {
+          required unsigned long width;
+          required unsigned long height;
+          required boolean hasHdrCapabilities;
```

Comment:
nit: name this hdrSupported

This spec generally talks about things (eme, codecs, configs) in terms of "support". For other info booleans we don't include a 'is' or 'has' prefix - supported implies "is supported". (hasAlphaChannel is a bit of an exception, but my handwavy reason is it doesn't have a suffix like 'supported' to imply its meaning...)